src/content/docs/release-notes/agent-release-notes/php-release-notes/php-agent-10-2-0-314.mdx

```diff
@@ -0,0 +1,20 @@
+---
+subject: PHP agent
+releaseDate: '2022-10-03'
+version: 10.2.0.314
+downloadLink: 'https://download.newrelic.com/php_agent/archive/10.2.0.314'
+---
+## New Relic PHP Agent v10.2.0.314
+
+### New Features
+* newrelic-daemon: Informational messages will be logged if the number of connected applications falls below 25 or exceeds 200. [#534](https://github.com/newrelic/newrelic-php-agent/pull/534)
+
+### Bug Fixes
+* agent: Fixed SIGSEGV in Drupal 7 drupal_http_request instrumentation for PHP 7.4, 8.0 and 8.1. [#538](https://github.com/newrelic/newrelic-php-agent/pull/538)
+* agent: Fixed displaying newrelic.application_logging*.enabled value in php info output. [#535](https://github.com/newrelic/newrelic-php-agent/pull/535)
```

Comment:
```suggestion
* agent: Fixed displaying `newrelic.application_logging*.enabled` value in PHP info output. [#535](https://github.com/newrelic/newrelic-php-agent/pull/535)
```

Reply:
Not sure if this should be in a codeblock

Reply:
@akristen Yes, it makes sense to be a code block. Thank you for applying this formatting.

Reply:
@akristen and @misiekn, I added the code formatting.